mobile/src/main/AndroidManifest.xml

```diff
@@ -10,6 +10,8 @@
     <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
     <uses-permission android:name="android.permission.NFC" />
     <uses-permission android:name="android.permission.WAKE_LOCK" />
+    <uses-permission android:name="android.permission.INSTALL_SHORTCUT"/>
```

Comment:
This one doesn't seem to be a thing though.

Reply:
If I'm searching for it, I find a few messengers that use this permission:
https://github.com/signalapp/Signal-Android/blob/master/AndroidManifest.xml#L72
https://github.com/DrKLO/Telegram/blob/master/TMessagesProj/src/main/AndroidManifest.xml#L43
and maybe WhatsApp as well: https://duckduckgo.com/?q=%22android.permission.INSTALL_SHORTCUT%22&t=ffcm&ia=web

Reply:
*shrug* I grepped the Android framework code (which includes support libs) for that string, and while I could find references to the com.android.launcher one, I couldn't find this one.

Reply:
I'm also not convinced e.g. Signal actually needs this permission either. The commit that introduced it seems rather ... random: https://github.com/signalapp/Signal-Android/commit/df674719597226485e1d3ecbd5b320e02ce1f797

Reply:
I tested this with API 21 and 25 emulators and both didn't need the `android.permission.INSTALL_SHORTCUT` permission. I'll remove it.